src/util/simplify_utils.cpp

```diff
@@ -119,30 +119,31 @@ static const struct saj_tablet &sort_and_join(
   return saj_table[i];
 }
 
-bool sort_and_join(exprt &expr)
```

Comment:
! This needs a new name

Reply:
Why?

Reply:
Because it no longer necessarily sorts! A person reading `sort_and_join(x, false);` might reasonably suppose that regardless of what that `false` means, `x` is certainly now sorted.